CHANGES.md

```diff
@@ -1,5 +1,15 @@
 # Release Notes
 
+## 0.12.0
+
+This release adds support for using placeholders in the `scie.lift.base` lift manifest value as well
+as the corresponding `SCIE_BASE` runtime control env var. A new placeholder is exposed in support of
+establishing custom scie base `nce` cache directories that respect the target OS user cache dir
+structure in the form of `{scie.user.cache_dir=<fallback>}`. Typically, this placeholder will expand
+to `~/Library/Caches` on macOS, `~\AppData\Local` on Windows and `~/.cache` on all other Unix
+systems unless over-ridden via OS-specific means or else unavailable for some reason, in which case
+the supplied `<fallback>` is used.
```

Comment:
I think I've been mis-reading this a few times. 

> unless over-ridden via OS-specific means or else unavailable for some reason, in which case the supplied `<fallback>` is used.

Using Linux as an example:

Without `SCIE_BASE` we default to `~/.cache`, and if that is unavailable - we use the `fallback` (if provided)?

With `SCIE_BASE=/tmp/myfolder`, we attempt to use `/tmp/myfolder`, and if that is unavailable, we use `fallback` (if provided)?

Am I understanding this correctly?

Reply:
Maybe, but your example is off. What you describe is true off the custom base value (specified via config or env var) uses the `{scie.user.cache_dir=<fallback>}` placeholder in it's value string somewhere.

Reply:
You must fill in a value in place of fallback. See the test: https://github.com/a-scie/jump/pull/137#discussion_r1317427848

If you don't supply a fallback that's a hard error. The `dirs::cache_dir()` API is fallible. I'm not sure why, but that implies calculating a value can fail; this is exactly why I force you to supply a fallback value when using the placeholder.